src/domain/entities/TransactionTypes.ts

```diff
@@ -1,7 +1,7 @@
 export enum TransactionKind {
   registration = "limited-partnership-registration",
   transition = "limited-partnership-transition",
-  postTransition = "limited-partnership-post-transition" // TODO confirm this is correct value
+  postTransition = "default"
```

Comment:
Seems a bit of an odd name. The old one was certainly more descriptive. Did it have to change?